content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Cut Service Mesh Overhead with Istio Ambient Mesh"
+description: An anaylsis of how cloud resources are utilized in ambient and sidecar service mesh architectures.
+publishdate: 2023-06-15
+attribution: "Greg Hanson - Solo.io"
+keywords: [istio,ambient,waypoint,ztunnel]
+---
+
+Istio Ambient Mesh is the new Istio data plane without requiring sidecars, refer to the [announcement](/blog/2022/introducing-ambient-mesh/) for more details. Istio Ambient Mesh is included in [Istio 1.18](https://github.com/istio/istio/releases/tag/1.18.0) but is still considered to be in [alpha feature status](/docs/releases/feature-stages/#feature-phase-definitions).
+
+Sidecars have been a staple of Istio’s architecture since day one and are responsible for the majority of features available in Istio today. However, sidecars require the injection of an additional container to each Kubernetes pod resource, each of which needs allocated resources from the pod.
+
+Many of you are familiar with the simplified operation brought by the Istio ambient architecture. Let’s explore how ambient mesh cuts the service mesh costs typically associated with sidecars.
+
+## Check Out the Savings
+
+Our test scenario deploys four instances of the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo), each to their own namespace, with each underlying microservice scaled to 3 replicas. There will be a total of 48 application services and 144 applications pods in this setup. Test scenarios utilize the provided [loadgenerator](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/src/loadgenerator) microservice running in each of the namespaces to generate traffic during the observed window. The tests deployed Istio in several scenarios and resource usage and allocation are compared across the runs. The scenarios are:
+
+- Istio with sidecar
+- Ambient with L4 ztunnel only
+- Ambient with L4 ztunnel and L7 waypoint proxies (one per namespace, scaled to 3 replicas)
+
+Spoilers! There are significant savings across the board for all ambient scenarios when compared to sidecars – up to 99% savings in usage and 98% in allocation.
+
+{{< image width="100%"
+    link="savings.png"
+    caption="Total Memory and CPU: Consumption and Allocation"
+    >}}
+
+Looking at total CPU and memory consumption, we have to remember that in the [ambient architecture](/blog/2022/introducing-ambient-mesh/), there are not sidecar containers for every application pod in the mesh. The result is that memory usage of Istio’s dataplane in the ztunnel-only ambient scenario uses *less than 1%* of what is used in sidecar scenarios, and still only 12% when waypoints are added. Looking at CPU, ztunnel once again uses 18% of what the sidecar scenario requires, and 60% when waypoints are deployed.
```

Comment:
suggest to talk about allocation first then resource usage as the above paragraph was about allocation so it transitions better.